module/sample_linux.json

```diff
@@ -0,0 +1,44 @@
+{
+  "components": [
+    {
+      "model": "viam:lidar:rplidar",
```

Comment:
[note to self] if you're referencing a modular resource, you need to give it the full name like we do here. if you're referencing a builtin resource, you give just the short name like we do [here](https://github.com/viamrobotics/viam-orb-slam3/pull/29/files#diff-868c9c2fdc104b26b1d16968030f23a5b552546a5bb86fbda780057b847cc41aR10).

i'm not sure if 'namespace' (L6 below) is related in any way.